lib/patientFinder/findPatientsIDs.rb

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/ruby
+require 'rubygems'
+require 'nokogiri'
+require 'health-data-standards'
+require 'zip/zipfilesystem'
+require 'fileutils'
+require 'health-data-standards'
+
+#a simple patient from QRDA - contains the source files name and the parsed record associated
+class PatientFromQRDA
```

Comment:
Agreed - my assumption was that Ruby is rather liberal with this - I need to read something on Ruby coding style